test/recase_test/sentence_case_test.exs

```diff
@@ -17,6 +17,15 @@ defmodule Recase.SentenceCaseTest do
     assert convert("sentence?!case") == "Sentence case"
   end
 
+  test "should return single letter" do
```

Comment:
Technically this falls out of the scope of this PR, but I thought it was good to be consistent across tests for this scenario.